test/validator/samples/a11y-html-has-lang/input.html

```diff
@@ -0,0 +1,5 @@
+<html lang="en"></html>
+<html lang="en-US"></html>
+<html lang={language}></html>
```

Comment:
~~Could you change `{language}` to `{{language}}` please?~~ never mind, just realised I can do it in the web editor!